src/cls/log/cls_log_client.h

```diff
@@ -23,8 +23,11 @@ void cls_log_list(librados::ObjectReadOperation& op, utime_t& from, utime_t& to,
 
 void cls_log_trim(librados::ObjectWriteOperation& op, const utime_t& from_time, const utime_t& to_time,
                   const string& from_marker, const string& to_marker);
+
+#ifndef CLS_CLIENT_HIDE_IOCTX
```

Comment:
could you please include the comment from cls_rgw_client.h above these #ifndefs in each file?
> // these overloads which call io_ctx.operate() should not be called in the rgw.
// rgw_rados_operate() should be called after the overloads w/o calls to io_ctx.operate()